packages/react/test/specs/components/TextArea/TextArea-test.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react'
+import { ReactWrapper } from 'enzyme'
+import { mountWithProvider as mount } from 'test/utils'
+import TextArea from 'src/components/TextArea/TextArea'
+import { isConformant } from 'test/specs/commonTests'
+
+const getTextAreaDomNode = (textAreaComp: ReactWrapper): HTMLTextAreaElement =>
+  textAreaComp.find('textarea').getDOMNode() as HTMLTextAreaElement
+
+const setUserTextAreaValue = (textAreaComp: ReactWrapper, value: string) => {
+  textAreaComp.find('textarea').simulate('change', { target: { value } })
+}
+
+const testValue = 'test value'
+
+describe('TextArea', () => {
+  describe('conformance', () => {
+    isConformant(TextArea, {
+      eventTargets: {
+        onChange: 'textarea',
+        onKeyDown: 'textarea',
+        onKeyPress: 'textarea',
+        onKeyUp: 'textarea',
+      },
+    })
+  })
+
+  describe('auto-controlled', () => {
```

Comment:
General thing, in SUIR it was common to structure tests in this way and the same thing is covered in our docs:
https://github.com/stardust-ui/react/blob/master/.github/test-a-feature.md#writing-tests

```tsx
describe('TextArea', () => {
  isConformant(TextArea)

  describe('defaultValue', () => {
     test('sets "defaultValue" as initial "value"', () => {
       const wrapper = mount(<TextArea defaultValue="foo" />)
       
       expect(wrapper.prop('value')).toBe('foo')
     })
  })

  describe('onChange', () => {
    test('is called with (e, data)', () => {
      const onChange = jest.fn()
      const wrapper = mount(<TextArea onChange={onChange} />)

      wrapper.simulate('change', { target: { value: 'foo' })
      expect(onChange).toBeCalledTimes(1)
      expect(onChange).toBeCalledWith(expect.objectContaining({ type: 'change' }), expect.objectContaining({ value: 'foo' })
    })
  })
})
```